src/main/java/com/conveyal/gtfs/loader/Feed.java

```diff
@@ -65,44 +58,51 @@ public Feed (DataSource dataSource, String tablePrefix) {
         routes = new JDBCTableReader(Table.ROUTES, dataSource, tablePrefix, EntityPopulator.ROUTE);
         stops = new JDBCTableReader(Table.STOPS, dataSource, tablePrefix, EntityPopulator.STOP);
         trips = new JDBCTableReader(Table.TRIPS, dataSource, tablePrefix, EntityPopulator.TRIP);
-//        shapePoints = new JDBCTableReader(Table.SHAPES, dataSource, tablePrefix, EntityPopulator.SHAPE_POINT);
```

Comment:
I realize this has been commented out for a while since @landonreed committed https://github.com/conveyal/gtfs-lib/commit/9244ff072cc6aa85bf6d6d315afa7861efdb10d4. But why? Should we add it back?

Reply:
Perhaps we should. It might be worth revisiting #130.